kotlin/scene-layer/README.md

```diff
@@ -0,0 +1,20 @@
+# Scene Layer
+Add a scene layer to a scene.
+
+![Scene Layer App](scene-layer.png)
+
+## How to use the sample
+Simply run the app.
+
+## How it works
+1. Create an `ArcGISScene` and set a `Basemap` with `ArcGISScene.setBasemap()`.
+1. Create a `SceneView` and set the scene to the view with `SceneView.setScene(...)`. 
+1. Create a `SceneLayer` and add it to the scene as an operational layer with `ArcGISScene.getOperationalLayers().add(sceneLayer)`.
```

Comment:
this should be the second step. 
```
Create an `ArcGISScene` and set a `Basemap` with `ArcGISScene.setBasemap()`.
Create a `SceneLayer` and add it to the scene as an operational layer with `ArcGISScene.getOperationalLayers().add(sceneLayer)`.
Create a `SceneView` and set the scene to the view with `SceneView.setScene(...)`. 
```

Also I think we should mention that we are setting the ViewPointCamera on the SceneView.

Reply:
apparently there is a newer readme version for all platforms that I was meant to use.